### DISCUSSION TITLE
Thoughts on vad

### DISCUSSION THREAD

**Discussion by YLQY** [snakers4/silero-vad#160] [Show and tell]
In our experiments, we found that when using vad for speech recognition, it is more inclined to disconnect the audio without cropping the audio. This reduces miscuts of vad. In our Chinese online game scene and TV drama scene, the effect of the former will be 3-4 points better than the latter :)

**Comment by snakers4** [comment]
Hi,

> when using vad for speech recognition, it is more inclined to disconnect the audio without cropping the audio. This reduces miscuts of vad.

Can you please elaborate on this? What is the difference between cropping and disconnecting? Do you have poor performance on edges?

Maybe provide a sample audio + hyper-params + the sample chart that is drawn using `visualize_probs=True`?
